src/ageml/ui.py

```diff
@@ -184,6 +290,289 @@ def load_csv(self, file_type):
         else:
             return None
 
+    def load_features(self, required=False):
+        """Load features from csv file.
+
+        Parameters
+        ----------
+        required: boolean to check if file is required
+        
+        Returns
+        -------
+        df: dataframe with features"""
+
+        # Load data
+        df = self.load_csv('features')
+
+        # Check that file
+        if required and df is None:
+            raise ValueError("Features file must be provided.")
+        elif df is None:
+            return df
+        
+        # Check that age column is present
+        if "age" not in df:
+            raise KeyError("Features file must contain a column name 'age', or any other case-insensitive variation.")
+
+        # Set features flag
+        self.flags['features'] = True
+
+        return df
+
+    def load_covariates(self, required=False):
+        """Load covariates from csv file.
+
+        Parameters
+        ----------
+        required: boolean to check if file is required
+        
+        Returns
+        -------
+        df: dataframe with covariates"""
+
+        # Load data
+        df = self.load_csv('covariates')
+
+        # Check required
+        if required and df is None:
+            raise ValueError("Covariates file must be provided.")
+        elif df is None:
+            return df
+        
+        # Set covariate flag
+        self.flags['covariates'] = True
+
+        # Set covariate for analysis
+        if hasattr(self.args, 'covar_name') and self.args.covar_name is not None:
+            self.flags['covarname'] = True
+            self.args.covar_name = self.args.covar_name.lower()
+            if self.args.covar_name not in df:
```

Comment:
Should this be:   `if self.args.covar_name not in df.columns`?
We are not checking if it is inside the columns, but inside the dataframe. Does it work?